tests/lib/eslint.js

```diff
@@ -959,6 +959,21 @@ describe("eslint", function() {
             sinon.assert.calledTwice(spyLiteral);
             sinon.assert.calledOnce(spyBinaryExpression);
         });
+
+        it("SourceCode objects should be processed", function() {
+            var config = {
+                rules: {},
+                ecmaFeatures: {}
+            };
+
+            eslint.reset();
+            eslint.verify(code, config, filename, true);
+            var sourceCode = eslint.getSourceCode();
```

Comment:
Yes, I didn't think about `eslint.parse()`, which I see now you are using in the test for `source-code.js`.  That's a much better way.